### DISCUSSION TITLE
Facturas a credito

### DISCUSSION THREAD

**Discussion by Jhamnerx** [thegreenter/greenter#224] [Show and tell]
Buenos dias, tengo un incoveniente al generar un factura al credito con decimal alguno tiene una manera de controlar esto?
 como ven en la imagen el total se pasa por .01 y obtengo el error de sunat que la suma de las cuotas es diferente al total.
![image](https://github.com/thegreenter/greenter/assets/30221273/a9f5512e-237f-4d25-b469-e120eecc11f1)

**Comment by giansalex** [comment]
hola, cuantos decimales deseas enviar?

**Reply by Jhamnerx** [reply]
segun sunat no puedo enviar mas de 2 decimales en los totales de las cuotas, el error es al calcular las cuotas siempre me da error por un decimal inexacto al total neto

**Reply by giansalex** [reply]
si, al parecer no se da el formato de 2 decimales, temporalmente puedes hacerlo en codigo, antes de setear el valor.

**Comment by giansalex** [comment]
Esta corregido en este commit https://github.com/thegreenter/greenter/commit/5240429dc0b19332ac2f34ede2aafa29627cf284

**Reply by Jhamnerx** [reply]
mi duda por ejemplo en el monto 1.280
si hago el calculo en 3 cuotas me da 426.67 x 3 = 1,280.01
sunat me va a rechazar porque no concide la suma de las cuotas con el monto a pagar.
ese es un problema con decimales, quiza alguien tiene una forma de hacerlo

**Reply by giansalex** [reply]
deberia haber un margen de tolerancia, cual es el codigo de error que te da?

**Reply by Jhamnerx** [reply]
![image](https://github.com/thegreenter/greenter/assets/30221273/5aa1a9f3-68e4-468c-bb22-1505c7c07578)
la suma de las cuotas debe ser igual al monto neto pendiente de pago

**Reply by giansalex** [reply]
una cuota tendría que ser `426.66` para que de el monto
